gdspy/__init__.py

```diff
@@ -173,6 +173,10 @@ def __init__(self, points, layer=0, datatype=0, verbose=True):
                 RuntimeWarning,
                 stacklevel=2)
         self.layer = layer
+
+        if isinstance(points[0][0], list):
```

Comment:
I'm not sure this test is a good idea... In general, the library assumes the user passes correct data types. There's no type checking in most methods. It would be probably better to use type annotations in the future than doing manual type checking everywhere.